bin/sl-meshctl.txt

```diff
@@ -84,6 +84,10 @@ services using `ls` or `status`.
         a grace period for any existing connections to finish. "Hard" stops
         kill the supervisor and its workers with `SIGTERM`.
 
+  tracing-start SVC       Restart all workers with tracing on.
```

Comment:
yah, that was my intention. altough it accepts SVC.EXEC I didnt want to complicate the help info